src/main/java/com/codingParkFun/rmsbackend/model/Semester.java

```diff
@@ -0,0 +1,86 @@
+package com.codingParkFun.rmsbackend.model;
+
+import java.sql.Date;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Semester {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	Long id;
+
+	String name;
+	String annualYear;
+	Date startDate;
+	Date endDate;
+	Boolean isActive;
+	Long schoolId;
```

Comment:
To mention relation_ship between semester and schoolId.

 @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "school_id")
    @JsonIgnore
    private School schoolId;

Reply:
As per my understanding, we need to mention relation in case if I need to fetch my Semester along with my school object. But in our case, we are keeping separate API for both.